sub_workflows/kf_VEP99_sub_wf.cwl

```diff
@@ -1,6 +1,6 @@
 cwlVersion: v1.0
 class: Workflow
-id: kf_VEP99_sub_wf
+id: kf_vep99_sub_wf
```

Comment:
are we trying to do keep all id in lower case with underscores? seems like that's not consistence with input variable names. maybe i've been paranoid on this.

Reply:
that's a good point, i should do that, otherwise extra hyphens get put in the tool name

Reply:
oh, now that i re-read, i am trying to keep the id names lower case because when you upload to cavatica, it adds an extra `-`. a separate issue from var names

Reply:
maybe some of this style stuff can be made into another ticket?